src/current/v25.2/create-logical-replication-stream.md

```diff
@@ -20,18 +20,30 @@ If the table you're replicating does not contain [user-defined types]({% link {{
 
 ## Required privileges
 
-`CREATE LOGICAL REPLICATION STREAM` requires one of the following privileges:
+`CREATE LOGICAL REPLICATION STREAM` creates a one-way LDR stream only. To achieve bidirectional replication, you must manually create two separate streams, one in each direction, with the required privileges set on both clusters.
 
-- The [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
-- The [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+LDR from cluster A to B represents a one-way stream from a source to a destination cluster. LDR from cluster B to A is the reverse stream for a bidirectional setup.
 
-Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+{% include_cached new-in.html version="v25.2" %} To run the `CREATE LOGICAL REPLICATION STREAM` statement to create an LDR stream, the following privileges are required:
+
+LDR direction | Cluster | User role | Required privilege 
+----------------------+---------+-----------+--------------------
+A ➔ B | A | User in the LDR connection string. | `REPLICATIONSOURCE`
+A ➔ B | B | User running the command. | `REPLICATIONDEST`
+B ➔ A | B | User in the LDR connection string. | `REPLICATIONSOURCE`
+B ➔ A | A | User running the command. | `REPLICATIONDEST`
+
+Grant a table-level privilege with the [`GRANT`]({% link {{ page.version.version }}/grant.md %}) statement to a [user or a role]({% link {{ page.version.version }}/security-reference/authorization.md %}#users-and-roles):
```

Comment:
nit: a user can still grant these privs at the system level if they'd like. If a user was running LDR over a 1000 tables, perhaps they'd rather use the system level priv as oppose to granting the priv on each table.

Reply:
Changed!